scripts/package.ts

```diff
@@ -106,6 +107,68 @@ function getVersionSuffix(feature: string, debug: boolean): string {
     return `${debugSuffix}${featureSuffix}${commitSuffix}`
 }
 
+/**
+ * @returns true if curl is available
+ */
+function isCurlAvailable(): boolean {
+    try {
+        child_process.execFileSync('curl', ['--version']);
+        return true;
+    } catch {
+        return false;
+    }
+}
+
+/**
+ * Small utility to download files.
+ */
+function downloadFiles(urls: string[], outputDir: string, outputFile: string): void {
+    if (platform() !== 'linux') {
+        return;
+    }
+
+    if (!isCurlAvailable()) {
```

Comment:
The `downloadFiles` function uses `curl` without validating the URLs. This could potentially lead to security issues if the URLs are not from trusted sources[^1].
```suggestion
function downloadFiles(urls: string[], outputDir: string, outputFile: string): void {
    if (platform() !== 'linux') {
        return;
    }

    if (!isCurlAvailable()) {
        return;
    }

    // Create output directory if it doesn't exist
    if (!nodefs.existsSync(outputDir)) {
        nodefs.mkdirSync(outputDir, { recursive: true });
    }

    const allowedDomains = ['raw.githubusercontent.com']; // Add other trusted domains as needed

    urls.forEach(url => {
        const parsedUrl = new URL(url);
        if (!allowedDomains.includes(parsedUrl.hostname)) {
            console.error(`Skipping download from untrusted domain: ${parsedUrl.hostname}`);
            return;
        }

        const filePath = path.join(outputDir, outputFile || '');

        try {
            child_process.execFileSync('curl', ['-o', filePath, url]);
        } catch (error) {
            console.error(`Failed to download file from ${url}: ${error}`);
        }
    })
}
```